src/NuGet.Core/NuGet.Protocol/Plugins/PluginManager.cs

```diff
@@ -317,11 +317,6 @@ private void Initialize(IEnvironmentVariableReader reader,
 #else
             _rawPluginPaths = reader.GetEnvironmentVariable(EnvironmentVariableConstants.CorePluginPaths);
 #endif
-            if (string.IsNullOrEmpty(_rawPluginPaths))
```

Comment:
Can you help me understand why we're changing the location of this call from PluginManager to PluginDiscoverer?

Reply:
This change was made as per https://github.com/NuGet/NuGet.Client/pull/5990#discussion_r1759318390 conversation.